kinto/core/storage/testing.py

```diff
@@ -24,8 +24,8 @@ def setUp(self):
         self.modified_field = "last_modified"
         self.client_error_patcher = None
 
-        self.record = {"foo": "bar"}
-        self.storage_kw = {"collection_id": "test", "parent_id": "1234", "auth": "Basic bWF0OjI="}
+        self.object = {"foo": "bar"}
```

Comment:
More generally, I don't love `self.object`, especially if we're using `obj` to avoid the builtin `object`.